common/FileUtil.cpp

```diff
@@ -318,7 +318,7 @@ namespace FileUtil
         if (access(path, W_OK) == 0)
             return true;
 
-        LOG_DBG("Cannot access path [" << path << "]: " << strerror(errno));
+        LOG_ERR("Cannot access path [" << path << "]: " << strerror(errno));
```

Comment:
This isn't an error. The purpose of this utility helper is to check whether a directory is writable or not. There is no expectation that the directory is writable. The DBG is just for that, debugging (to help devs understand which directories are writable and which aren't, to understand the decision made by the code).